examples/reference/elements/bokeh/Sankey.ipynb

```diff
@@ -37,14 +37,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "hv.Sankey([\n",
+    "sankey = hv.Sankey([\n",
     "    ['A', 'X', 5],\n",
     "    ['A', 'Y', 7],\n",
     "    ['A', 'Z', 6],\n",
     "    ['B', 'X', 2],\n",
     "    ['B', 'Y', 9],\n",
     "    ['B', 'Z', 4]]\n",
-    ").options(width=600, height=400)"
+    ")\n",
+    "sankey.opts(opts.Sankey(width=600, height=400))"
```

Comment:
This is two keywords, but I'd argue width/height is so common this should also be inlined.